starlette/tasks.py

```diff
@@ -0,0 +1,30 @@
+import uuid
```

Comment:
We *might?* want to call it `starlette/background.py` to conflict a little less with asyncio tasks?